packages/emotion-theming/types/tsconfig.json

```diff
@@ -1,16 +1,23 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "es2015",
-    "declaration": true,
-    "strict": true,
-    "moduleResolution": "node",
+    "baseUrl": "../",
+    "forceConsistentCasingInFileNames": true,
     "jsx": "react",
-    "lib": ["es6"],
+    "lib": [
+      "es6"
+    ],
+    "module": "commonjs",
+    "noEmit": true,
     "noImplicitAny": true,
     "noImplicitThis": true,
+    "strict": true,
```

Comment:
You can clean it up a bit now because of strict mode. You can remove noImplicitAny, noImplicitThis, strictNullChecks and strictFunctionTypes.

Reply:
I usually left them for clarity. Does this verbosity bother you?

Reply:
It is a habit of mine to keep them as concise as possible. But that's just me. I'm leaving it up to you 👍